plugin/torch/torch_criterion-inl.h

```diff
@@ -61,7 +60,7 @@ class TorchCriterionOp : public Operator {
     int err = lua_pcall(L, 0, 1, 0);
     CHECK_EQ(err, 0) << lua_tostring(L, -1);
     // serialize
-    TorchState::Serialize(&chunk_);
+    this->lua_reference_ = lua_ref(L, LUA_REGISTRYINDEX);
```

Comment:
Since we are not sharing lua states any more, can we just aways keep the torch module in L? Do we have to get it out?